components/ErrorEventsDashboard.js

```diff
@@ -1,42 +1,44 @@
+import { ExternalLinkAlt } from '@styled-icons/fa-solid/ExternalLinkAlt'
 import moment from 'moment'
-import { Button } from 'react-bootstrap'
-import useSWR, { mutate } from 'swr'
+import { useState } from 'react'
+import useSWR from 'swr'
 
-import FetchMessage from './FetchMessage'
+import PageControls from './PageControls'
 
 const ERROR_EVENTS_URL = `${process.env.API_BASE_URL}/api/admin/bugsnag/eventsummary`
 
 function ErrorEventsDashboard () {
-  const result = useSWR(ERROR_EVENTS_URL)
-  const { data, error } = result
-  const events = data && data.data
-  const hasEvents = events && events.length > 0
-  const MAX_EVENTS_TO_DISPLAY = 100
+  const [offset, setOffset] = useState(0)
+  const limit = 40
```

Comment:
Use 50 or 25 as a limit (it produces a nice 50, 100, 150... sequence for pagination).